src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.behavior.Shape;
+
+public class Circle implements Shape {
```

Comment:
all our figures should implement interfaces `Drawable` and `AreaCalculator` - how can we optimise the code so that we won’t have to write `implements Drawable, AreaCalculator` for Circle and other child figures?